tests/foreman/api/test_repository.py

```diff
@@ -452,65 +558,87 @@ def test_positive_create_same_name_different_orgs(self):
 
         :CaseLevel: Integration
         """
-        repo1 = entities.Repository(product=self.product).create()
-        repo2 = entities.Repository(name=repo1.name).create()
-        self.assertEqual(repo1.name, repo2.name)
+        org_2 = entities.Organization().create()
+        product_2 = entities.Product(organization=org_2).create()
+        repo_2 = entities.Repository(product=product_2, name=repo.name).create()
+        assert repo_2.name == repo.name
 
     @pytest.mark.tier2
-    def test_positive_create_puppet_repo_same_url_different_orgs(self):
+    @pytest.mark.parametrize(
+        'repo_options',
+        **parametrized(
+            [{'content_type': 'puppet', 'url': 'https://omaciel.fedorapeople.org/7c74c2b8/'}]
+        ),
+        indirect=True,
+    )
+    def test_positive_create_puppet_repo_same_url_different_orgs(self, repo_options, repo):
         """Create two repos with the same URL in two different organizations.
 
         :id: 7c74c2b8-732a-4c47-8ad9-697121db05be
 
+        :parametrized: yes
+
         :expectedresults: Repositories are created and puppet modules are
             visible from different organizations.
 
         :CaseLevel: Integration
         """
-        url = 'https://omaciel.fedorapeople.org/7c74c2b8/'
-        # Use setup product for the first repo and create new org/product
-        # for the second one
-        org = entities.Organization().create()
-        products = (self.product, entities.Product(organization=org).create())
-        # Create repositories within different organizations/products
-        for product in products:
-            repo = entities.Repository(url=url, product=product, content_type='puppet').create()
-            repo.sync()
-            self.assertGreaterEqual(repo.read().content_counts['puppet_module'], 1)
+        repo.sync()
+        assert repo.read().content_counts['puppet_module'] >= 1
+
+        # create new org/product/repo
```

Comment:
```suggestion
        # create new org, product, and repo
```
Please don't use slash in text

Reply:
Fixed.